docs/templates.md

```diff
@@ -34,7 +34,7 @@ const template = [
 A custom post type can register its own template during registration:
 
 ```php
-function register_post_type() {
+function my_template_to_books() {
```

Comment:
Good catch on this conflicting with the core `register_post_type` function name. To better reflect that this is registering a post type, though, maybe it should be named as such, while still avoiding the conflict?

Thinking something along the lines of...

```php
function register_book_post_type() {}
```

```php
function myplugin_register_post_types() {}
```

Thoughts?

Reply:
`myplugin_register_book_post_type` would be best.

Since this is PHP code in the global namespace, it's best practice to use a vendor prefix in the function. Using unique function prefixes or namespaces is required by dot org theme and plug-in directories.